web-common/src/features/dashboards/url-state/DashboardURLStateSync.svelte

```diff
@@ -13,22 +13,22 @@
   import type { HTTPError } from "@rilldata/web-common/runtime-client/fetchWrapper";
   import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
 
+  export let metricsViewName: string;
+  export let exploreName: string;
   export let defaultExplorePreset: V1ExplorePreset;
+  export let initExploreState: Partial<MetricsExplorerEntity>;
+  export let initUrlSearch: string;
   export let partialExploreState: Partial<MetricsExplorerEntity>;
+  export let urlSearchForPartial: string;
```

Comment:
If I'm following correctly, `partialExploreState` is actually the `urlPartialExploreState`. If so, we shouldn't be passing both `urlPartialExploreState` and `urlSearchForPartial`. We should not pass two props that are "synonyms" (different representations of the same state) – we should pass just one of them.